jetson/percep/pcl.hpp

```diff
@@ -50,20 +50,31 @@ class compareLine {
 class PCL {
     public:
 
-    //Constructor
-    PCL() : 
-        bearing{0}, distance{0}, detected{false},
-        pt_cloud_ptr{new pcl::PointCloud<pcl::PointXYZRGB>} {
-
-        #if ZED_SDK_PRESENT
-        sl::Resolution cloud_res = sl::Resolution(PT_CLOUD_WIDTH, PT_CLOUD_HEIGHT);
-        cloudArea = cloud_res.area();
-        #else
-        cloudArea = PT_CLOUD_WIDTH*PT_CLOUD_HEIGHT;
-        #endif
+    //Constants
+    int MAX_FIELD_OF_VIEW_ANGLE;
+    int PT_CLOUD_WIDTH;
+    int PT_CLOUD_HEIGHT;
+    int HALF_ROVER;
+    double UP_BD_Z;
+    double UP_BD_Y;
+    double LOW_BD ;
+    double ROVER_W_MM;
+    float LEAF_SIZE;
+    //RANSAC constants
+    int MAX_ITERATIONS;
+    double SEGMENTATION_EPSLION;
+    double DISTANCE_THRESHOLD;
+    //Euclidean cluster constants
+    int CLUSTER_TOLERANCE;
+    int MIN_CLUSTER_SIZE;
+    int MAX_CLUSTER_SIZE;
 
```

Comment:
There seemed to be some compiler warnings about the order that you initialize these values. Could you get rid of those warnings? Should just have to do with reordering the variable definitions.